features/product/managing_product_reviews/browsing_product_reviews.feature

```diff
@@ -0,0 +1,19 @@
+@managing_product_reviews
+Feature: Browsing product reviews
+    In order to know customer's opinion about my products
+    As an Administrator
+    I want to browse product reviews
+
+    Background:
+        Given the store has customer "Mike Ross" with email "ross@teammike.com"
+        And the store has a product "Lamborghini Gallardo Model"
+        And this product has a review titled "Awesome" and rated 5 with a comment "Nice product" added by customer "ross@teammike.com"
+        And this product has also a review titled "Bad" and rated 1 with a comment "Really bad" added by customer "ross@teammike.com"
```

Comment:
We should add another product and assign the bad review to it (we will validate that reviews for all products are displayed), since now the background looks really strange, love&hate 😆